cmake/example_cmake_find_gtsam/main.cpp

```diff
@@ -69,7 +68,7 @@ int main(int argc, char** argv) {
   // 2a. Add a prior on the first pose, setting it to the origin
   // A prior factor consists of a mean and a noise model (covariance matrix)
   noiseModel::Diagonal::shared_ptr priorNoise = noiseModel::Diagonal::Sigmas(Vector3(0.3, 0.3, 0.1));
-  graph.emplace_shared<PriorFactor<Pose2> >(1, Pose2(0, 0, 0), priorNoise);
+  graph.addPrior<>(1, Pose2(0, 0, 0), priorNoise);
```

Comment:
Why the <> ? Does it not compile without?

Reply:
I mistakenly thought the <> were still necessary even for deduced template arguments. I've replaced all `addPrior<>` with `addPrior`.